src/Tyrannoport/Program.cs

```diff
@@ -1,15 +1,30 @@
-﻿using System.Threading.Tasks;
+﻿using System.Linq;
+using System.Threading.Tasks;
+using DocoptNet;
 
 namespace Tyrannoport
 {
     class Program
     {
+        private const string Usage = @"Tyrannoport
+
+    Usage:
+        tyrannoport [options] <trx>...
```

Comment:
Probably want at least an `--output` option to specify an output file or directory.